winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,654 @@
+using Syncfusion.UI.Xaml.DataGrid.Export;
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+async void Save(MemoryStream stream, string filename)
+{
+    StorageFile stFile;
+
+    if (!(Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.Phone.UI.Input.HardwareButtons")))
+    {
+        FileSavePicker savePicker = new FileSavePicker();
+        savePicker.DefaultFileExtension = ".xlsx";
+        savePicker.SuggestedFileName = filename;
+        savePicker.FileTypeChoices.Add("Excel Documents", new List<string>() { ".xlsx" });
+        var hwnd = System.Diagnostics.Process.GetCurrentProcess().MainWindowHandle;
+        WinRT.Interop.InitializeWithWindow.Initialize(savePicker, hwnd);
+        stFile = await savePicker.PickSaveFileAsync();
+    }
+    else
+    {
+        StorageFolder local = Windows.Storage.ApplicationData.Current.LocalFolder;
+        stFile = await local.CreateFileAsync(filename, CreationCollisionOption.ReplaceExisting);
+    }
+    if (stFile != null)
+    {
+        using (IRandomAccessStream zipStream = await stFile.OpenAsync(FileAccessMode.ReadWrite))
+        {
+            //Write compressed data from memory to file
+            using (Stream outstream = zipStream.AsStreamForWrite())
+            {
+                byte[] buffer = stream.ToArray();
+                outstream.Write(buffer, 0, buffer.Length);
+                outstream.Flush();
+            }
+        }
+        //Launch the saved Excel file
+        await Windows.System.Launcher.LaunchFileAsync(stFile);
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> SfDataGrid exports data to excel by using [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). You can refer [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating exported work sheets. 
+
+## Excel exporting options
+
+Exporting operation can be customized by passing `DataGridExcelExportOptions` instance as argument to `ExportToExcel` method. 
+
+### Export mode
+
+By default, actual value only will be exported to excel. If you want to export the display text, you need to set `ExportMode` property as `Text`. 
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.ExportMode = ExportMode.Text;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Export groups with outlines
+
+By default, all the groups in dataGrid will be exported in expanded state. You can enable outlines in excel based on groups by setting the `ShowOutlines` property as `true` in `DataGridExcelExportOptions`.
```

Comment:
ShowOutlines property exported the groups based on expanded state right? If yes, use the below.

You can enable outlines in excel based on group expanded state by setting the `ShowOutlines` property

Reply:
modified